api/v1/kataconfig_types.go

```diff
@@ -100,7 +100,7 @@ func init() {
 // KataInstallationStatus reflects the status of the ongoing kata installation
 type KataInstallationStatus struct {
 	// InProgress reflects the status of nodes that are in the process of kata installation
-	InProgress KataInstallationInProgressStatus `json:"inprogress,omitempty"`
+	InProgress KataInstallationInProgressStatus `json:"inProgress,omitempty"`
```

Comment:
Is it safe to change this with respect to operator upgrade ?

Reply:
that's a good point, for this PR in general. We are changing the API. Do we need to create a new api version? Does it need a webhook that converts from the old to the new version?

Reply:
I didn't know it was changing API. I only thought about consistency with other fields following camel case. As it is such a small change I can revert it back.